wire-library/wire-runtime/src/commonMain/kotlin/com/squareup/wire/WireField.kt

```diff
@@ -66,7 +66,14 @@ annotation class WireField(
     REPEATED,
     ONE_OF,
     /** Implies [REPEATED]. */
-    PACKED;
+    PACKED,
+    /**
+     * Special label to defines proto3 fields which should not be emitted if their value is equal
+     * to their type's respective identity value. E.g.: a field of type `int32` will not get emitted
+     * if its value is `0`.
+     */
+    IDENTITY_IF_ABSENT,
```

Comment:
The doc describes "absent if identity" which is a property of encoding and not "identity if absent" which would be a property of decoding. Should this be named `ABSENT_IF_IDENTITY`?

Reply:
This actually go both ways, right. I think all those labels can be read for both encoding and decoding. Maybe just `IDENTITY`?

Reply:
Ah, I see.

`IGNORE_IDENTITY`? `OMIT_IDENTITY`?

Reply:
I like `OMIT_IDENTITY`